WooCommerce/Classes/ViewRelated/CardPresentPayments/WCPayCardBrand+icons.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import Yosemite
+
+extension WCPayCardBrand {
+    /// Icon that represents the card brand in SVG format
+    ///
+    var icon: UIImage? {
+        UIImage(named: iconName)
+    }
+
+    /// Icon name to be used for the card icon UIImage
+    var iconName: String {
+        switch self {
```

Comment:
I used the switch so that we'd provide a placeholder image even if we add a new card brand without providing an image for it in future... so I think I'll leave this one if that's OK 😊